tools/astarte_e2e/lib/astarte_e2e/config.ex

```diff
@@ -0,0 +1,97 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2020 Ispirata Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule AstarteE2E.Config do
+  use Skogsra
+
+  alias AstarteE2E.Config.SchedulerOption
+
+  @envdoc "Astarte Pairing URL."
```

Comment:
We could support both options: (1) configuration of Astarte URL, (2) explicit setting of both pairing and websocket URL.

(1) would simplify the configuration and easily fit standard deployments.
(2) would allow flexibility in the config. Setting the variables in this case would override the option of (1).

Any opinions? @bettio @rbino